pysparkling/sql/functions.py

```diff
@@ -577,6 +592,7 @@ def expr(expression):
     return parse(expression)
 
 
+# noinspection PyShadowingBuiltins
```

Comment:
The line seems to have been mistakenly duplicated

Reply:
I'll remove this.